.github/workflows/merge-gatekeeper.yml

```diff
@@ -23,4 +23,4 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           timeout: 3600
           interval: 30
```

Comment:
we can include all the hardcoded values at one place as mentioned [here](https://github.com/kedro-org/kedro-viz/pull/1876/files#r1585617759). May be we can have a separate ticket for this

Reply:
Same as above [comment](https://github.com/kedro-org/kedro-viz/pull/1876#discussion_r1586152387).